STYLE_API_TEMPLATE.md

```diff
@@ -116,22 +145,44 @@ Include a table with these columns:
 Field | Data Type | Description
 :--- | :--- | :--- 
 
-#### Sample request
+#### Sample response
```

Comment:
Remove this section - this is section #6 in the new structure.

Reply:
Removed this last section, "Sample response."